client/blocks/comments/comment-actions.jsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+import { changeCommentStatus } from 'state/comments/actions';
+import CommentLikeButtonContainer from './comment-likes';
+import CommentApproveAction from './comment-approve-action';
+import EllipsisMenu from 'components/ellipsis-menu';
+import PopoverMenuItem from 'components/popover/menu-item';
+import PopoverMenuSeparator from 'components/popover/menu-separator';
+
+const CommentActions = ( {
+	post,
+	comment: { isPlaceholder, status },
+	translate,
+	activeEditCommentId,
+	activeReplyCommentID,
+	commentId,
+	handleReply,
+	onReplyCancel,
+	approveComment,
+	unapproveComment,
+	trashComment,
+	spamComment,
+	editComment,
+	editCommentCancel,
+ } ) => {
+	const showReplyButton = post && post.discussion && post.discussion.comments_open === true;
+	const showCancelReplyButton = activeReplyCommentID === commentId;
+	const showCancelEditButton = activeEditCommentId === commentId;
+	const isApproved = status === 'approved';
+
+	// Only render actions for non placeholders
+	if ( isPlaceholder ) {
+		return null;
+	}
+
+	return (
+		<div className="comments__comment-actions">
+			{ showReplyButton &&
+				<button className="comments__comment-actions-reply" onClick={ handleReply }>
+					<Gridicon icon="reply" size={ 18 } />
+					<span className="comments__comment-actions-reply-label">{ translate( 'Reply' ) }</span>
+				</button>
+			}
+			{ showCancelReplyButton &&
+				<button className="comments__comment-actions-cancel-reply" onClick={ onReplyCancel }>{ translate( 'Cancel reply' ) }</button>
+			}
+			{ showCancelEditButton &&
+				<button className="comments__comment-actions-cancel-reply" onClick={ editCommentCancel }>{ translate( 'Cancel' ) }</button>
```

Comment:
There's a few long lines here, might be good to wrap the attributes of the `<button>`s here.